osu.Game/Graphics/UserInterface/BreadcrumbControl.cs

```diff
@@ -15,14 +15,15 @@ namespace osu.Game.Graphics.UserInterface
     public class BreadcrumbControl<T> : OsuTabControl<T>
     {
         private const float padding = 10;
-        private const float item_chevron_size = 10;
 
-        protected override TabItem<T> CreateTabItem(T value) => new BreadcrumbTabItem(value)
+        protected virtual float ItemChevronSize => 10;
```

Comment:
why is this even at `BreadcrumbControl`? The only usage has you passing it to a class you are constructing. So why not just specify it within that class?

Reply:
Strip size (which is in `BreadcrumbControl`) depends on the icon size

Reply:
That strip is not preset in `OverlayBreadcrumbControl`, but might affect possible future implementations

Reply:
I've fixed this for you. But then in continued review, it doesn't even match the god damn design. And this part isn't required for it to match.

Care to put in a bit more effort? There should be no underline on the breadcrumb display.

Reply:
Once you told me that ingame elements should be consistent. I mean, when once I tried to implement custom links, which matched web design - you told me to use already implemented. That's why I didn't touch this control much.

Reply:
If they are to be consistent, the line should be removed from all instances since at the moment it looks like a bug. But at that point why even have two classes?